paloesg/excide
# Add view and download documents to motif document repo

# Description

- Add view document for motif and download button for each document
- Motif PDF documents will not be converted to 
- Add iframe for PDF viewer and activestorage-office-previewer gem to preview office file
- For now, the code defaults to viewing only, meaning right click cannot save image or document. Also, iframe toolbar is disabled! 

## Remarks

Current only works for images as pdf has some issues relating to image magick and pdf to img conversion. Security for pdf could also be an issue as its using the url of the file.
--------------------------------------------  24/09/20 update ------------------------------------------------------------
Access with permission control not integrated yet. 

# Testing

- Add a document via Symphony document repo upload and view using motif/documents
- Check that document cannot be downloaded via right click or by iframe toolbar
